test/email_validator_test.dart

```diff
@@ -116,15 +116,10 @@ void main() {
     'θσερ@εχαμπλε.ψομ', // Greek
   ];
 
-  test('Validate that validate throws error on null email', () {
-    expect(() => EmailValidator.validate(null),
-        throwsA(new isInstanceOf<ArgumentError>()));
-  });
-
   test('Validate invalidAddresses are invalid emails', () {
     invalidAddresses.forEach((String actual) => expect(
         EmailValidator.validate(actual, true), equals(false),
-        reason: 'E-mail: ' + actual));
+        reason: 'E-mail: ' + actual.toString()));
```

Comment:
This `toString` should not be needed since `actual` is non-nullable `String`.

Reply:
Yeah, I was playing around with the ? operator and forgot to remove that 👍